obj/createTask.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+    //DOM elements
+    submitButton : '[type="submit"]',
+    taskName : '[id="taskDetailsForm.name"]',
+    taskDescription : '[id="taskDetailsForm.description"]',
+    taskBill : '.Select-value',
+    taskBillSelect : '[aria-label="?"]',
+    taskBillSelected : '[aria-selected="true"]',
```

Comment:
This might not work as expected and will find only first selected value on the page. Is there any other more stable way to find it?